styles.css

```diff
@@ -0,0 +1,47 @@
+body {
+  background-color: aliceblue;
+}
```

Comment:
You should always include at least one generic family name in a font-family list since there's no guarantee that any given font is available.

in this case, family-font was not declared at all.